dimscord/objects.nim

```diff
@@ -589,7 +590,7 @@ proc updateMessage*(m: Message, data: JsonNode): Message =
         attachments = data{"attachments"}.getElems.map(newAttachment)
         embeds = data{"embeds"}.getElems.mapIt(it.`$`.fromJson(Embed))
     if result.referenced_message.isSome and "referenced_message" in data:
-        result.referenced_message = some data["referenced_message"].newMessage
+        result.referenced_message = some data["referenced_message"].newMessage(nil)
```

Comment:
Why nil?

Reply:
I made it in an hurry in order to pass `nim check`, it will be fixed when I have more free time :)